<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportWindow</class>
 <widget class="QWidget" name="ImportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>961</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="m_importJSON">
    <widget class="QPushButton" name="importer_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>410</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Importer</string>
     </property>
    </widget>
    <widget class="QLabel" name="dropLabel_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>361</width>
       <height>201</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="l_importXLSX">
    <widget class="QLabel" name="dropLabel">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>361</width>
       <height>201</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="importer">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>290</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Importer</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
